hw/ip/kmac/dv/kmac_masked_sim_cfg.hjson

```diff
@@ -14,5 +14,6 @@
   en_build_modes: ["enable_mask_mode"]
 
   // exclusion files
-  vcs_cov_excl_files: ["{proj_root}/hw/ip/kmac/dv/cov/kmac_masked_unr_excl.el"]
+  vcs_cov_excl_files: ["{proj_root}/hw/ip/kmac/dv/cov/kmac_masked_unr_excl.el",
+                       "{proj_root}/hw/ip/kmac/dv/cov/kmac_masked_cov_excl.el"]
```

Comment:
Why is this file repeated? Same below.

Reply:
sorry, I'm not sure I follow, which file is repeated here? The `{variant}_unr_excl.el` file is meant for exclusions auto-generated by the UNR flow, and the `{variant}_cov_excl.el` is meant for any manual exclusions.